Ошибка: HTTPConnectionPool(host='109', port=80): Max retries exceeded with url: / (Caused by NameResolutionError("<urllib3.connection.HTTPConnection object at 0x000002027ABC55A0>: Failed to resolve '109' ([Errno 11001] getaddrinfo failed)"))